org.eclipse.scout.rt.uinotification/src/main/java/org/eclipse/scout/rt/api/uinotification/UiNotificationPutOptions.java

```diff
@@ -36,7 +48,41 @@ public Boolean getTransactional() {
     return m_transactional;
   }
 
+  /**
+   * @param publishOverCluster
+   *          Whether a notification is published to all other cluster nodes or not. Default is {@code true}.
+   */
+  public UiNotificationPutOptions withPublishOverCluster(Boolean publishOverCluster) {
+    m_publishOverCluster = publishOverCluster;
+    return this;
+  }
+
+  public Boolean getPublishOverCluster() {
+    return m_publishOverCluster;
+  }
+
+  /*
+   * factory methods
+   */
+
+  /**
```

Comment:
What is the benefit of this create method compared to just using the constructor? I don't think this is necessary.